notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T231-analysis.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>It looks like the links are not being rendered properly. It can be that you have a single space after the first link definition (<code>[LVV-T231]</code>). Try adding an extra space and it will probably render it better.</p><p> </p><p>Please, add a summary of what the test is doing and what are the expected results. The idea is that someone opens this notebook and have minimal information to understand what is going on. If they want more details, then they access the links you provided.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added and fixed.